zone/client.cpp

```diff
@@ -410,8 +411,12 @@ Client::~Client() {
 		zone->ClearEXPModifier(this);
 	}
 
-	if(IsInAGuild())
-		guild_mgr.SendGuildMemberUpdateToWorld(GetName(), GuildID(), 0, time(nullptr));
+	if (!IsZoning()) {
```

Comment:
Any reason to not do `if (!IsZoning() && IsInAGuild())` to consolidate the conditions?

Reply:
I was just thinking that there will be more items that could be moved within based on the !zoning criteria, though at the moment no other reason really.